src/templates/Header/Header.tsx

```diff
@@ -0,0 +1,66 @@
+import { Grid, Link, Typography, useMediaQuery } from '@mui/material';
+import styles from './Header.module.scss';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+
+export function Header() {
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const tablet: boolean = useMediaQuery(theme.breakpoints.up('tablet'));
+  const isRegister = false;
+
+  const REGISTER_IDX_START = 2;
+  const LOGIN_IDX_START = 0;
+  const TABLET_IDX_ADJUST = 0;
+  const MOBILE_IDX_ADJUST = 1;
+
+  const headerRegisterJsonIdx = isRegister ? REGISTER_IDX_START : LOGIN_IDX_START;
+  const headerTabletJsonIdx = tablet ? TABLET_IDX_ADJUST : MOBILE_IDX_ADJUST;
+  const jsonIdx = headerRegisterJsonIdx + headerTabletJsonIdx;
+  const jsonSubscript = `.${String(jsonIdx)}`;
```

Comment:
Header component should take `universityName`, `dormitoryName` and `faculty`, so we can reuse this component across `RegisterPage` and `LoginPage`

Reply:
resolved in f29528d